include/ocpp/v201/smart_charging.hpp

```diff
@@ -28,9 +30,16 @@ enum class ProfileValidationResultEnum {
     ChargingProfileExtraneousStartSchedule,
     ChargingSchedulePeriodsOutOfOrder,
     ChargingSchedulePeriodInvalidPhaseToUse,
+    ChargingSchedulePeriodUnsupportedNumberPhases,
+    ChargingSchedulePeriodExtraneousPhaseValues,
     DuplicateTxDefaultProfileFound
 };
 
+inline std::ostream& operator<<(std::ostream& os, const ProfileValidationResultEnum validation_result) {
```

Comment:
I would suggest to keep the implementation in the cpp file for this. Also it would be nice if we actually print the result in a human readable form. Take a look at the `v201/enums.hpp` for how it is done there (that is autogenerated btw).